CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+- [Local Setup](#-Setup-Without-Docker)
+- [Docker Setup](#-Setup-With-Docker-Compose)
+
+## ⚙ Setup Without Docker
+### Client
+- `cd client`
+- `npm install` to install all dependencies 
+- `npm start` to build & start the React app
+
+### Server
+- `cd server`
+- `npm install` in the root directory to install all dependencies
+- `npm start` to start the Node.js server
+
+### Database
+- Install PostgreSQL 12.4 (untick pgAdmin installation, we'll install this separately) https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+- pgAdmin version 4.27 release https://www.pgadmin.org/download/
+- open `pgAdmin` -> `PostgreSQL 12` -> `Databases`
+- Right click `Databases` and `Create` -> `Database`
+- name:`'postgres'`, owner: `'postgres'`, password: `123456` (This is the current hard coded config defined in server file)
+- If setup correctly you should see the following when running npm start in server:
+
+![Alt text](./pgconnected.png)
+
+## 🐳 Setup With Docker Compose
+
+### Server
+- Make sure that you have `.env` file on the root directory of server (ask one of us!). This file defines `DB_DATABASE`, `DB_PASSWORD`, `DB_USER`, and `DB_HOST`
+- `npm install` in the root directory of server to install all dependencies
+- run `docker-compose up`
+- The output of `docker-compose up` should look similar to:
+```bash
```

Comment:
This can maybe go now that the output must be inconsistent with what we have now!

Reply:
Agree with removing screenshots :). Resources also seem unnecessary.